modules/nw-autoscaling-ingress-controller.adoc

```diff
@@ -0,0 +1,222 @@
+// Module included in the following assemblies:
+//
+// * networking/ingress-controller-configuration.adoc
+
+:_content-type: PROCEDURE
+[id="nw-autoscaling-ingress-controller_{context}"]
+= Autoscaling an Ingress Controller
+
+Automatically scale an Ingress Controller to dynamically meet routing performance or availablility requirements such as the requirement to increase throughput. The following procedure provides an example for scaling up the default `IngressController`.
+
+.Prequisites
+. You installed the OpenShift CLI (`oc`).
+. You have access to the cluster as a user with the `cluster-admin` role.
+. You installed Custom Metrics Autoscaler Operator.
+
+.Procedure
+. Create a project in the `openshift-ingress-operator` namespace by running the following command:
++
+[source,terminal]
+----
+$ oc project openshift-ingress-operator
+----
+
+. Enable OpenShift monitoring for user-defined projects by applying the following config map:
++
+[source,terminal]
+----
+$ oc apply -f - <<EOF
```

Comment:
The <<EOF in line 28 renders red. iirc it means like some spacing is off or something. It may be nothing, but just thought I'd mention it.

Reply:
Could it be `$ oc apply -f - <<EOF` is the command and the user applies 

```
apiVersion: v1
kind: ConfigMap
metadata:
  name: cluster-monitoring-config
  namespace: openshift-monitoring
data:
  config.yaml: |
    enableUserWorkload: true
EOF
```

after that command is run?

Reply:
This is a way of [applying a config from an interactive shell](https://superuser.com/a/1003761), but I'll separate it out into two steps: one for the config creation, and one for the application of the new config. They're equivalent, but I think your suggestion to use the two separate steps is more probably more readable 🙂